src/sections/Home/Proud-maintainers/proudMaintainers.style.js

```diff
@@ -150,7 +150,7 @@ const ProjectItemWrapper = styled.section`
 		}
       @media screen and (max-width: 550px) {
           img {
-            padding: 1.65rem 0;
+            padding: 1.65rem 0.5rem 1.65rem 0.8rem;
           }
           h4 {
             font-size: 0.7rem;
```

Comment:
Move the 550px media query after the 660px media query, otherwise the latter is overriding the former.

Reply:
Done, kindly check